composer.json

```
@@ -24,20 +24,20 @@
     "contao-community-alliance/dc-general": "~2.1",
     "contao-community-alliance/meta-palettes": "^1.11 || ^2.0",
     "contao/core-bundle": "^4.4",
-    "doctrine/dbal": "^2.6",
+    "doctrine/dbal": "^2.6 || ^3.3",
```

```suggestion
    "doctrine/dbal": "^2.11 || ^3.3",
```

The BC layer for the new methods were introduced in `doctrine/dbal 2.11` , so allowing `^2.6` may break installations